lib/active_record_shards/connection_switcher.rb

```diff
@@ -7,6 +7,10 @@ module ConnectionSwitcher
     class LegacyConnectionHandlingError < StandardError; end
     class IsolationLevelError < StandardError; end
 
+    Thread.attr_accessor :__active_record_shards__disallow_replica_by_thread,
+                         :_active_record_shards_in_migration,
+                         :_shard_selection
```

Comment:
Could we use 1 naming pattern instead of 3? 😅 E.g.
```suggestion
    Thread.attr_accessor :_active_record_shards_disallow_replica_by_thread,
                         :_active_record_shards_in_migration,
                         :_active_record_shards_shard_selection
```

Reply:
I thought about changing that but maybe there's a reason why the double underscores was added? Maybe it's meant to indicate something (tbh I'm not sure what though 😅 )

Reply:
But I'll go ahead and change them.